### PR TITLE
Update Remove-CsAutoAttendant.md

### PR DESCRIPTION
**Pull request by CLYVR**
Adding note to indicate that resource accounts must be removed first.

### CODE REVIEW

**Comment by CLYVR** [comment]
@get-itips @dariomws @tseward @tiburd - some minor corrects please and thanks.

**Review by get-itips — APPROVED** [review]

**Comment by tiburd** [comment]
@dariomws @tseward 
Please @ mention me after the last of you two review/approve this PR.
Thanks!

File: skype/skype-ps/skype/Remove-CsAutoAttendant.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Remove-CsAutoAttendant cmdlet to delete an Auto Attendant (AA).
 
+> [!NOTE]
+>Remove any associated resource accounts with [Remove-CsOnlineApplicationInstanceAssociation](Remove-CsOnlineApplicationInstanceAssociation.md) before attempting to delete the Auto Attendant (AA)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Remove-CsAutoAttendant.md:19]:
```suggestion
> Remove any associated resource accounts with [Remove-CsOnlineApplicationInstanceAssociation](Remove-CsOnlineApplicationInstanceAssociation.md) before attempting to delete the AA.
```

Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsAutoAttendant.md:19]:
```suggestion
> [!NOTE]
> Remove any associated resource accounts with [Remove-CsOnlineApplicationInstanceAssociation](Remove-CsOnlineApplicationInstanceAssociation.md) before attempting to delete the Auto Attendant (AA).
```

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CLYVR, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**